bindgen fails on Windows

I'm unable to build `libui-ffi`, here is the output
```plain
PS C:\Users\lucac\Documents\GitHub\libui> cargo build
    Blocking waiting for file lock on build directory
   Compiling libui-ffi v0.3.0 (C:\Users\lucac\Documents\GitHub\libui\libui-ffi)
error: failed to run custom build command for `libui-ffi v0.3.0 (C:\Users\lucac\Documents\GitHub\libui\libui-ffi)`

Caused by:
  process didn't exit successfully: `C:\Users\lucac\Documents\GitHub\libui\target\debug\build\libui-ffi-3da00774548ee8e3\build-script-build` (exit code: 101)
  --- stderr
  C:\Program Files\Microsoft Visual Studio\2022\Community\VC\Tools\MSVC\14.34.31933\include/vadefs.h:61:33: error: expected ';' after top level declarator
  C:\Program Files\Microsoft Visual Studio\2022\Community\VC\Tools\MSVC\14.34.31933\include/vcruntime.h:96:1: error: expected identifier
  C:\Program Files\Microsoft Visual Studio\2022\Community\VC\Tools\MSVC\14.34.31933\include/vcruntime.h:96:20: error: expected ';' after top level declarator
  C:\Program Files\Microsoft Visual Studio\2022\Community\VC\Tools\MSVC\14.34.31933\include/vcruntime.h:193:29: error: expected ';' after top level declarator
  C:\Program Files\Microsoft Visual Studio\2022\Community\VC\Tools\MSVC\14.34.31933\include/vcruntime.h:194:30: error: typedef redefinition with different types ('__int64' (aka 'unsigned int') vs 'long long')
  C:\Program Files\LLVM\lib\clang\15.0.6\include/stddef.h:35:26: note: previous definition is here
  C:\Program Files\Microsoft Visual Studio\2022\Community\VC\Tools\MSVC\14.34.31933\include/vcruntime.h:386:51: error: unknown type name 'uintptr_t'
  C:\Program Files\Microsoft Visual Studio\2022\Community\VC\Tools\MSVC\14.34.31933\include/vcruntime.h:387:67: error: unknown type name 'uintptr_t'
  C:\Program Files\Microsoft Visual Studio\2022\Community\VC\Tools\MSVC\14.34.31933\include/vcruntime.h:391:8: error: unknown type name 'uintptr_t'
  C:\Program Files\Microsoft Visual Studio\2022\Community\VC\Tools\MSVC\14.34.31933\include/vcruntime.h:399:1: error: a parameter list without types is only allowed in a function definition     
  C:\Program Files\Microsoft Visual Studio\2022\Community\VC\Tools\MSVC\14.34.31933\include/vcruntime.h:399:18: error: expected ';' after top level declarator
  ./libui/ui.h:99:2: error: type name requires a specifier or qualifier
  ./libui/ui.h:99:14: error: type specifier missing, defaults to 'int'; ISO C99 and later do not support implicit int [-Wimplicit-int]
  ./libui/ui.h:129:12: error: unknown type name 'uintptr_t'
  thread 'main' panicked at libui-ffi\build.rs:57:10:
  Unable to generate bindings: ClangDiagnostic("C:\\Program Files\\Microsoft Visual Studio\\2022\\Community\\VC\\Tools\\MSVC\\14.34.31933\\include/vadefs.h:61:33: error: expected ';' after top level declarator\nC:\\Program Files\\Microsoft Visual Studio\\2022\\Community\\VC\\Tools\\MSVC\\14.34.31933\\include/vcruntime.h:96:1: error: expected identifier\nC:\\Program Files\\Microsoft Visual Studio\\2022\\Community\\VC\\Tools\\MSVC\\14.34.31933\\include/vcruntime.h:96:20: error: expected ';' after top level declarator\nC:\\Program Files\\Microsoft Visual Studio\\2022\\Community\\VC\\Tools\\MSVC\\14.34.31933\\include/vcruntime.h:193:29: error: expected ';' after top level declarator\nC:\\Program Files\\Microsoft Visual Studio\\2022\\Community\\VC\\Tools\\MSVC\\14.34.31933\\include/vcruntime.h:194:30: error: typedef redefinition with different types ('__int64' (aka 'unsigned int') vs 'long long')\nC:\\Program Files\\Microsoft Visual Studio\\2022\\Community\\VC\\Tools\\MSVC\\14.34.31933\\include/vcruntime.h:386:51: error: unknown type name 'uintptr_t'\nC:\\Program Files\\Microsoft Visual Studio\\2022\\Community\\VC\\Tools\\MSVC\\14.34.31933\\include/vcruntime.h:387:67: error: unknown type name 'uintptr_t'\nC:\\Program Files\\Microsoft Visual Studio\\2022\\Community\\VC\\Tools\\MSVC\\14.34.31933\\include/vcruntime.h:391:8: error: unknown type name 'uintptr_t'\nC:\\Program Files\\Microsoft Visual Studio\\2022\\Community\\VC\\Tools\\MSVC\\14.34.31933\\include/vcruntime.h:399:1: error: a parameter list without types is only allowed in a function definition\nC:\\Program Files\\Microsoft Visual Studio\\2022\\Community\\VC\\Tools\\MSVC\\14.34.31933\\include/vcruntime.h:399:18: error: expected ';' after top level declarator\n./libui/ui.h:99:2: error: type name requires a specifier or qualifier\n./libui/ui.h:99:14: error: type specifier missing, defaults to 'int'; ISO C99 and later do not support implicit int [-Wimplicit-int]\n./libui/ui.h:129:12: error: unknown type name 'uintptr_t'\n")
  note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

I had the same problem with another project and I solved by adding the following:
```rust
let bindings = BindgenBuilder::default()
    ...
    .clang_args([
        "-target", &target_triple,
    ])
    ...
```
See <https://github.com/rust-lang/rust-bindgen/issues/1760>

I have no idea why this is happening, a few months ago the project built fine, I don't know what changed in the meantime. Anyway, I think that this would fix the issue.

--- a/libui-ffi/build.rs
+++ b/libui-ffi/build.rs
@@ -49,6 +49,9 @@ fn main() {
         .header("wrapper.h")
         .opaque_type("max_align_t") // For some reason this ends up too large
         //.rustified_enum(".*")
+        .clang_args([
+            "-target", &target_triple,
+        ])
         .trust_clang_mangling(false) // clang sometimes wants to treat these functions as C++
         .generate()
         .expect("Unable to generate bindings");

